assets/js/components/settings/SettingsActiveModule/ConfirmDisconnect.js

```diff
@@ -16,11 +16,6 @@
  * limitations under the License.
  */
 
-/**
- * External dependencies
- */
-import PropTypes from 'prop-types';
```

Comment:
@danielgent why did you delete prop types for this component? Could you, please, add it back?

Reply:
Apolgies @eugene-manuilov .  I managed to remove them in a merge :thinking:   No idea what happened there! They're added back now :heavy_check_mark: